### ISSUE TITLE
`README` outdated

### ISSUE THREAD

**Issue by BracketJohn** [peopledoc/procrastinate#364]
Hey guys,

love the idea, been testing it out a little for use at work. I tried to follow the examples from the `README.md`, as they showed programmatic usage in comparison to the CLI based approach I was able to find in your documentation.

Sadly, the examples do not work (out of the box):
```py
# Somewhere in your program, run a worker asynchronously
worker = procrastinate.Worker(
    app=app,
    queues=["sums"]
)
```

Fails, since `Worker` is not exported by `procrastinate`.

This:
```py
await worker.run_async()
```

Fails, since `AttributeError: 'Worker' object has no attribute 'run_async'`.

This is not urgent to me, I've been able to work around it, but I think for a quick try out it would be nice if `README` examples work (:

Thanks for the library!

**Comment by elemoine** [comment]
@BracketJohn thanks for the report, we'll look into it!

**Comment by BracketJohn** [comment]
Thanks for addressing this so quickly! However, the `async` example is still broken (`AttributeError: module 'procrastinate' has no attribute 'Worker'`).

Instead of just reporting, I decided to go through both the `sync` and `async` and opened a PR with fixes (: I hope that that helps, but ofc. do not expect any action based on the PR. For an outsider like me, the changes would be helpful.

In doing so, I also noticed that the `schema_manager` seems to be `sync` only and that that does not play along nicely with the `async` stream of procrastinate. I've filed #367  for that.

The PR I opened is #368, looking forward to your review.
